doc-content/enterprise-only/springboot/bus-app-auth_con.adoc

```diff
@@ -0,0 +1,39 @@
+[id='bus-app-auth_{context}']
+= Business application authentication and authorization
+
+By default, business applications are secured by protecting all REST endpoints
+(URLs that contain `/rest/`). In addition, business applications have two sets of log in credentials that allow users to connect to  Business Central or Decision Central in development mode: the user with the ID `user` and password `user`  and the  `kie-server` user (password `kieserver1!`).
```

Comment:
kieserver user

Reply:
Thankjs @MarianMacik. I have updated the repositories section and correct kie-server (misread your comment earlier). Also, I have now made two books - one for PAM and one for DM - just in case you missed the comment in the jira (links there).